src/go/rpk/pkg/cli/cloud/mcp/mcp.go

```diff
@@ -45,55 +47,68 @@ func newInstall(fs afero.Fs, p *config.Params) *cobra.Command {
 		Short: "Install Redpanda Cloud MCP server",
 		Long: `Install Redpanda Cloud MCP server.
 		
-Only Claude Desktop is supported at this time.
-Writes an mcpServer entry with name "redpandaCloud" into claude_desktop_config.json in Claude Desktop's config directory.`,
+Supports Claude Desktop and Claude Code.
+Writes an mcpServer entry with name "redpandaCloud" into the appropriate config file.`,
 		Args: cobra.NoArgs,
 		Run: func(_ *cobra.Command, _ []string) {
 			cfg, err := p.Load(fs)
 			out.MaybeDie(err, "Failed to load rpk config: %w", err)
 
-			if mcpClient != "claude" {
+			var file string
+			switch mcpClient {
+			case "claude":
+				configDir, err := os.UserConfigDir()
+				out.MaybeDie(err, "failed to get user configuration directory: %w", err)
+				file = filepath.Join(configDir, "Claude", "claude_desktop_config.json")
+			case "claude-code":
+				homeDir, err := os.UserHomeDir()
+				out.MaybeDie(err, "failed to get user home directory: %w", err)
+				file = filepath.Join(homeDir, ".claude.json")
+			default:
 				out.Die("Unsupported client: %s", mcpClient)
 			}
 
-			configDir, err := os.UserConfigDir()
-			out.MaybeDie(err, "failed to get user configuration directory: %w", err)
-			file := filepath.Join(configDir, "Claude", "claude_desktop_config.json")
-
 			fileMode := fz.FileMode(0o600)
 			jsonStr := `{}`
 			f, err := os.OpenFile(file, os.O_RDONLY, 0o600)
 			if err != nil && !os.IsNotExist(err) {
-				out.MaybeDie(err, "failed to open Claude Desktop config file: %v", err)
+				out.MaybeDie(err, "failed to open config file: %v", err)
 			}
 			defer f.Close()
 
 			if err == nil {
 				stat, err := f.Stat()
-				out.MaybeDie(err, "failed to stat Claude Desktop config file: %v", err)
+				out.MaybeDie(err, "failed to stat config file: %v", err)
 				fileMode = stat.Mode()
 
 				bytez, err := io.ReadAll(f)
-				out.MaybeDie(err, "failed to read content of Claude Desktop config file: %v", err)
+				out.MaybeDie(err, "failed to read content of config file: %v", err)
 				jsonStr = string(bytez)
 			}
 
 			jsonStr, err = sjson.Set(jsonStr, "mcpServers.redpandaCloud.command", "rpk")
-			out.MaybeDie(err, "failed to patch Claude Desktop config: %v", err)
+			out.MaybeDie(err, "failed to patch config: %v", err)
 			var mcpArgs []string
 
 			if cfg.VirtualProfile().CloudEnvironment != "" {
 				mcpArgs = append(mcpArgs, "-X")
 				mcpArgs = append(mcpArgs, fmt.Sprintf("cloud_environment=%s", cfg.VirtualProfile().CloudEnvironment))
 			}
+
+			configDir, err := os.UserConfigDir()
+			out.MaybeDie(err, "failed to get user configuration directory: %w", err)
+
+			rpkConfigPath := filepath.Join(configDir, "rpk", "rpk.yaml")
+
```

Comment:
You can use https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/config/rpk_yaml.go#L29, which effectively is the same logic.

Reply:
done